integration-cli/docker_cli_commit_test.go

```diff
@@ -223,6 +223,13 @@ func (s *DockerSuite) TestCommitChange(c *check.C) {
 		"--change", "ENV DEBUG true",
 		"--change", "ENV test 1",
 		"--change", "ENV PATH /foo",
+		"--change", "LABEL foo bar",
+		"--change", "MAINTAINER Main Tainer <maintainer@test.com>",
```

Comment:
minor thing, but someone once told me you should use "example.com" due to possible legal ramifications.